src/App/components/Settings.tsx

```diff
@@ -119,16 +120,22 @@ export default function Settings(): JSX.Element {
         defaultDirectory={defaultAppBuilderDirectory}
       />
       <Card mb={3}>
-        <H5 mb={3}>Output</H5>
-        <FileSelector
-          buttonText="Save videos to..."
-          file={settings.outputDirectory}
-          options={{
-            title: 'Select Output Folder',
-            properties: ['openDirectory'],
-          }}
-          onFileSelected={settings.setOutputDirectory}
-        />
+        <H5 mb={2}>Output</H5>
+        <Flex alignItems="center">
+          <FileSelector
+            buttonText="Save videos to..."
+            file={settings.outputDirectory}
+            options={{
+              title: 'Select Output Folder',
+              properties: ['openDirectory'],
+            }}
+            onFileSelected={settings.setOutputDirectory}
+          />
+          <Tooltip content="Reset to default directory">
+              <Button minimal icon="reset" onClick={(): Function => settings.setOutputDirectory(DEFAULT_OUTPUT_DIRECTORY)} />
```

Comment:
I'd declare this arrow function as a `const` variable above and refer to the variable here. Naming the function makes it more readable.

Reply:
Done. I also positioned the button's tooltip underneath.